### PR TITLE
Add Elektron_Overbridge_2.0.37.3.dmg v2.0.37.3

### PR DESCRIPTION
**Pull request by crsrusl**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

`brew cask style --fix {{cask_file}}` reported 1 error as there is a mismatch between the 'url' and 'homepage', have added in comment line as per instructions.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by crsrusl** [comment]
The uninstaller opens a GUI which works perfectly, but its seems to hang the test script - any ideas how to resolve?

File: Casks/elektron-overbridge.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'elektron-overbridge' do
+  version :latest
+  sha256 :no_check
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:3]:
```
version '2.0.37.3'
sha '3f54c2300af5d86b021e4400d78cad9ac5aea1c4b6ba96814a123521b87c3feb'
```

File: Casks/elektron-overbridge.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  # https://www.elektron.se/overbridge/ was verified as official when first introduced to the cask
+  url 'https://s3-eu-west-1.amazonaws.com/se-elektron-devops/release/8d1ef129-6161-52ac-b65a-ea18489fd654/Elektron_Overbridge_2.0.37.3.dmg'
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:6]:
```
# https://s3-eu-west-1.amazonaws.com/se-elektron-devops/ was verified as official when first introduced to the cask
url "https://s3-eu-west-1.amazonaws.com/se-elektron-devops/release/8d1ef129-6161-52ac-b65a-ea18489fd654/Elektron_Overbridge_#{version}.dmg"
appcast 'https://www.elektron.se/support/?connection=overbridge#resources' 
```

File: Casks/elektron-overbridge.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  name 'Overbridge'
+  homepage 'https://www.elektron.se/overbridge/'
+
+  pkg 'Elektron Overbridge Installer 2.0.37.3.pkg'
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:10]:
```
pkg "Elektron Overbridge Installer #{version}.pkg"
```

File: Casks/elektron-overbridge.rb (hunk 1 above)
Comment (crsrusl) [reply Casks/elektron-overbridge.rb:3]: @miccal sorry, I'm a bit new to this - Do I need to make the changes suggested here or have you implemented them? Thanks, Chris

Comment (miccal) [reply Casks/elektron-overbridge.rb:3]: @crsrusl you need to make the changes, thanks.

File: Casks/elektron-overbridge.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+cask 'elektron-overbridge' do
+  version '2.0.37.3'
+  sha '3f54c2300af5d86b021e4400d78cad9ac5aea1c4b6ba96814a123521b87c3feb'
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:3]: Sorry, this should be `sha256`, not `sha`.

File: Casks/elektron-overbridge.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+  # https://s3-eu-west-1.amazonaws.com/se-elektron-devops/ was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:5]: Remove the `https://` from the comment.

File: Casks/elektron-overbridge.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  url "https://s3-eu-west-1.amazonaws.com/se-elektron-devops/release/8d1ef129-6161-52ac-b65a-ea18489fd654/Elektron_Overbridge_#{version}.dmg"
+  appcast 'https://www.elektron.se/support/?connection=overbridge#resources'
```
Comment (miccal) [inline Casks/elektron-overbridge.rb:7]: There is an extra space at the end.

**Comment by vitorgalvao** [comment]
From the website (emphasis mine):

> Overbridge is a free of charge software suite offering powerful tools and deep control **when connecting compatible Elektron instruments to computers**.

So this needs hardware to be useful, correct?

In that case, it needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks), but you’re already following them.

Please let me know if I’m wrong.

Thank you @miccal (as always) for the help in getting the cask good for submission.

**Pull request closed without merging**